archive/frames/views.py

```diff
@@ -154,13 +154,14 @@ def zip(self, request):
         return Response(request_serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
     @staticmethod
-    def _get_aggregate_values(query_set, field, aggregate_field):
+    def _get_aggregate_values(query_set, query_filters, field, aggregate_field):
+        if field in query_filters:
+            return [query_filters[field]]
```

Comment:
This just ensures we dont waste time doing an aggregate query when we are filtering by that specific term too.